pkg/services/cloudmigration/cloudmigrationimpl/snapshot_mgmt.go

```diff
@@ -57,7 +58,13 @@ func (s *Service) getMigrationDataJSON(ctx context.Context, signedInUser *user.S
 		})
 	}
 
+	softDeleteEnabled := s.features.IsEnabledGlobally(featuremgmt.FlagDashboardRestore)
+
 	for _, dashboard := range dashboards {
+		if softDeleteEnabled && !dashboard.Deleted.IsZero() {
```

Comment:
Could we test this? If it is not that hard

Reply:
Sure, I will add a test before I merge